### PR TITLE
Refactored taskman variables usage

### PR DESCRIPTION
**Pull request by costaconrado**
This PR changes the way Taskman uses global variables.

In order to improve variables management, some development environment variables were moved to .env file.
This PR also change taskman tests, using conftest to set a proxy during the tests if the HTTPS_TASKMAN_PROXY var is set.

### CODE REVIEW

File: apps/taskman/tests/conftest.py [hunk 1]
```diff
@@ -20,6 +20,12 @@ def enable_db_access_for_all_tests(db):
     pass
 
 
+@pytest.fixture(autouse=True)
+def enable_proxy_for_stage_environment(monkeypatch):
+    if HTTPS_TASKMAN_PROXY:
+        monkeypatch.setenv("HTTPS_PROXY", HTTPS_TASKMAN_PROXY)
```
Comment (osoukup) [inline apps/taskman/tests/conftest.py:26]: Shouldn't **"HTTPS_PROXY"** be **"HTTPS_TASKMAN_PROXY"**?

**Review by osoukup — APPROVED** [review]
LGTM

File: docs/developer/DEVELOP.md [hunk 2]
```diff
@@ -52,9 +52,9 @@ RH_CERT_URL="https://foo.bar"
 # otherwise all the writes are performed only locally in OSIDB
 BBSYNC_SYNC_TO_BZ=1
 
-# Enable https proxy (optional)
-# This variable is only required when using taskman to access Red Hat's stage environment
-HTTPS_PROXY="http://foo.bar"
+# Enable https proxy in taskman test cases (optional)
+# This variable is only used in case you access a Jira stage environment behind a proxy
+HTTPS_TASKMAN_PROXY="http://foo.bar"
```
Comment (Elkasitu) [inline docs/developer/DEVELOP.md:57]: This should say something like:

"Run taskman tests behind a proxy (optional)
 This variable is only necessary if rewriting the taskman cassettes locally and using the Stage Red Hat JIRA instance, which requires a proxy to be accessed."

File: docs/developer/DEVELOP.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ JIRA_AUTH_TOKEN=####################
 # Jira URL (optional, defaults to RedHat production Jira)
 JIRA_URL="https://foo.bar"
 
-# Task management Jira URL (optional, defaults to Red Hat stage Jira)
+# Task management Jira URL
 JIRA_TASKMAN_URL="https://foo.bar"
```
Comment (Elkasitu) [inline docs/developer/DEVELOP.md:28]: Unrelated, but we probably want both of these converged

**Review by Elkasitu — APPROVED** [review]

**Pull request merged**